src/constants.ts

```diff
@@ -15,9 +15,16 @@ export const DEFAULT_PIPELINE = "vib-pipeline.json"
 /**
  * Max waiting time for an execution graph to complete
  *
- * @default 90 minutes
+ * @default 2 hours
```

Comment:
This value does not match the documentation. Lets change it back to 90mins. That's ok.